src/compute/hash.rs

```diff
@@ -29,7 +30,8 @@ pub fn hash_primitive<T: NativeType + Hash>(array: &PrimitiveArray<T>) -> Primit
 }
 
 #[multiversion]
-#[clone(target = "x86_64+aes+sse3+ssse3+avx+avx2")]
+#[clone(target = "[x86|x86_64]+avx+avx2")]
```

Comment:
I also needed the features `ssse3` `sse3` and `aes`  at least (as those are used specifically by `aHash`